### PR TITLE
Update cheatsheet commands

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Summary
Zrzut ekranu 2022-09-20 o 12 56 31

Closes ENG-356.

## Test Plan
Tested manually.

## Checklist
- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/src/components/shell/TerminalSidebar.js [hunk 1]
```diff
@@ -465,26 +465,26 @@ function Cheatsheet() {
...
   {
-    command: 'bundle install <app> <bundle>',
-    description: 'Installs a bundle and writes the configuration to this installation\'s context.',
+    command: 'bundle install <repo> <bundle>',
+    description: <>E.g. <strong>plural bundle install dagster dagster-gcp</strong> would configure Dagster for deployment to GCP.</>,
   },
   {
     command: 'build',
-    description: 'Builds your workspace.',
+    description: 'Generates all the infrastructure as code.',
```
Comment (maciaszczykm) [inline www/src/components/shell/TerminalSidebar.js:480]: Updated according to @samweaver suggestion.

File: www/src/components/shell/TerminalSidebar.js [hunk 2]
```diff
@@ -465,26 +465,26 @@ function Cheatsheet() {
...
   },
   {
     command: 'deploy',
-    description: 'Deploys the current workspace. This command will sniff out git diffs in workspaces, topsort them, then apply all changes.',
+    description: 'Deploys all installed bundles.',
```
Comment (maciaszczykm) [inline www/src/components/shell/TerminalSidebar.js:484]: Updated according to @samweaver suggestion.

File: www/src/components/shell/TerminalSidebar.js [hunk 3]
```diff
@@ -465,26 +465,26 @@ function Cheatsheet() {
...
   },
   {
-    command: 'watch <app>',
+    command: 'watch <repo>',
```
Comment (maciaszczykm) [inline www/src/components/shell/TerminalSidebar.js:487]: Changed `<app>` to `<repo>` to align it in all places.

**Review by maciaszczykm — COMMENTED** [review]
`plural init` and `plural destroy` were not added.

cc @samweaver @michaeljguarino for review.

**Comment by samweaver** [comment]
overall looks good to me, ill let @michaeljguarino be the approver and comment on any gramatical enhanacements he may want to make! thanks marcin!

File: www/src/components/shell/TerminalSidebar.js [hunk 4]
```diff
@@ -465,26 +465,26 @@ function Cheatsheet() {
 function CheatsheetCommands() {
   const commands = [{
     command: 'repos list',
-    description: 'Lists available repositories to install.',
+    description: 'Shows all available applications on the platform (repos) and the cloud providers they can be deployed to (bundles).',
```
[the remark this answers is not in the document]
Comment (michaeljguarino) [reply www/src/components/shell/TerminalSidebar.js:468]: let's drop the `(bundles)` parenthetization, or maybe change it to `(using a bundle)`.  It makes it seem like a bundle is a cloud provider with the current phrasing

**Review by michaeljguarino — APPROVED** [review]
small nit, but otherwise lgtm

**Pull request merged**